test/wdio.conf.ts

```diff
@@ -91,20 +91,28 @@ export const config: Options.Testrunner = {
     //
     // Place inside the array to run sequentially.
     [
-      './test/specs/**/*.e2e.ts'
-
+      ...EnvironmentSettings.getInstance().specFiles
+      //
```

Comment:
Since this has been moved into the EnvironmentSettings, can we delete these here? Or add a comment if you think it might be confusing?

Reply:
I'll add a comment.  i like the ability of setting it here or within EnvironmentSettings.